UnitTests/ConsoleDrivers/KeyEventArgsTests.cs

```diff
@@ -0,0 +1,175 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Text;
+using Terminal.Gui.ConsoleDrivers;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Terminal.Gui.InputTests;
+public class KeyEventArgsTests {
+	readonly ITestOutputHelper _output;
+
+	public KeyEventArgsTests (ITestOutputHelper output)
+	{
+		_output = output;
+	}
+
+	[Fact]
+	public void Constructor_Default_ShouldSetKeyToUnknown ()
+	{
+		var eventArgs = new KeyEventArgs ();
+		Assert.Equal (Key.Unknown, eventArgs.Key);
+	}
+
+	[Theory]
+	[InlineData (Key.Enter)]
+	[InlineData (Key.Esc)]
+	[InlineData (Key.A)]
+	public void Constructor_WithKey_ShouldSetCorrectKey (Key key)
+	{
+		var eventArgs = new KeyEventArgs (key);
+		Assert.Equal (key, eventArgs.Key);
+	}
+
+	[Fact]
+	public void HandledProperty_ShouldBeFalseByDefault ()
+	{
+		var eventArgs = new KeyEventArgs ();
+		Assert.False (eventArgs.Handled);
+	}
+
+	[Fact]
+	public void KeyValueProperty_ShouldReturnCorrectIntValue ()
+	{
+		var eventArgs = new KeyEventArgs (Key.A);
+		Assert.Equal ((int)Key.A, eventArgs.KeyValue);
+	}
+
+	[Theory]
+	[InlineData (Key.AltMask, true)]
+	[InlineData (Key.A, false)]
+	public void IsAltProperty_ShouldReturnCorrectValue (Key key, bool expected)
+	{
+		var eventArgs = new KeyEventArgs (key);
+		Assert.Equal (expected, eventArgs.IsAlt);
+	}
+
+	// Similar tests for IsShift and IsCtrl
+	[Fact]
+	public void ToString_ShouldReturnReadableString ()
+	{
+		var eventArgs = new KeyEventArgs (Key.CtrlMask | Key.A);
+		Assert.Equal ("Ctrl+A", eventArgs.ToString ());
+	}
+
+	[Theory]
+	[InlineData (Key.CtrlMask | Key.A, '+', "Ctrl+A")]
+	[InlineData (Key.AltMask | Key.B, '-', "Alt-B")]
+	public void ToStringWithSeparator_ShouldReturnFormattedString (Key key, Rune separator, string expected)
+	{
+		Assert.Equal (expected, KeyEventArgs.ToString (key, separator));
+	}
+
+	[Theory]
+	[InlineData ((Key)'☑', "☑")]
+	[InlineData ((Key)'英', "英")]
+	[InlineData ((Key)'{', "{")]
+	//[InlineData ((Key)'\'', "\'")]
```

Comment:
With the changes above uncomment this should also pass.

Reply:
Yep!